pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -840,14 +847,14 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 	// containers cannot be specified on pod creation.
 	if utilfeature.DefaultFeatureGate.Enabled(features.EphemeralContainers) {
 		for _, idx := range podContainerChanges.EphemeralContainersToStart {
-			start("ephemeral container", ephemeralContainerStartSpec(&pod.Spec.EphemeralContainers[idx]))
+			start(metrics.EphemeralContainerLabel, ephemeralContainerStartSpec(&pod.Spec.EphemeralContainers[idx]))
```

Comment:
I think we need an `ACTION REQUIRED:` release note since these labels are changing.

Reply:
Sure, happy to do that. We could also keep the old format for the low, low price of more code. What do you think?

Reply:
Since we introduced snake case for the metric labels I reverted this change. Log messages will remain unchanged.